### ISSUE TITLE
Can't scroll in code window and output window

### ISSUE THREAD

**Issue by denismullaraj** [krzyzanowskim/OnlineSwiftPlayground#24]
Sometimes scrolling within the code window and output window does not work properly.

This is a screenrecording of the issue in the output window, couldn't replicate again the code window issue.

![ezgif com-video-to-gif (5)](https://user-images.githubusercontent.com/25382699/81315184-45411080-908a-11ea-9a04-eca1cb9c4f6d.gif)

Using Chrome Version 81.0.4044.129 (Official Build) (64-bit) on MacOS.
Code wrote in the Code Window to reproduce the issue in the Output Window:
```
import Foundation

class Adapter: NSObject {
    static var lat: Int = {
        // some calculations
        print("lat")
        return 1
    }()
}

print(Adapter.lat)
print(Adapter.lat)

class Adapter2: NSObject {
    static var width: Int {
        // some calculations
        print("width")
        return 1
    }
}

print(Adapter2.width)
print(Adapter2.width)
```

**Comment by milank-gintaa** [comment]
I have faced the same problem

**Comment by denismullaraj** [comment]
@milank-gintaa Yeah, I'm still facing the same problem, not sure if anyone is taking a look at it currently
